target/linux/ramips/base-files/etc/uci-defaults/04_led_migration

```diff
@@ -0,0 +1,67 @@
+#!/bin/sh
+
+. /lib/functions.sh
+. /lib/functions/migrations.sh
+
+board=$(board_name)
+
+case "$board" in
+fon,fonera-20n)
+	migrate_leds "^fonera20n:=fonera-20n:"
+	;;
+intenso,memory2move)
+	migrate_leds "^m2m:=memory2move:"
+	;;
+lenovo,newifi-y1)
+	migrate_leds "^y1:=newifi-y1:"
+	;;
+lenovo,newifi-y1s)
+	migrate_leds "^y1s:=newifi-y1s:"
+	;;
+netgear,wn3000rp-v3)
+	migrate_leds "^wn3000rpv3:=wn3000rp-v3:"
+	;;
+netgear,wndr3700-v5)
+	migrate_leds "^wndr3700v5:=wndr3700-v5:"
+	;;
+tplink,archer-c2-v1)
+	migrate_leds "^c2-v1:=archer-c2-v1:"
+	;;
+tplink,archer-c20-v1)
+	migrate_leds "^c20-v1:=archer-c20-v1:"
+	;;
+tplink,archer-c20-v4)
+	migrate_leds "^c20-v4:=archer-c20-v4:"
+	;;
+tplink,archer-c20i)
+	migrate_leds "^c20i:=archer-c20i:"
+	;;
+tplink,archer-c50-v1)
+	migrate_leds "^c50:=archer-c50-v1:"
+	;;
+tplink,archer-c50-v3)
+	migrate_leds "^c50-v3:=archer-c50-v3:"
+	;;
+tplink,archer-c50-v4)
+	migrate_leds "^c50-v4:=archer-c50-v4:"
+	;;
+tplink,archer-mr200)
+	migrate_leds "^mr200:=archer-mr200:"
+	;;
+youku,yk1)
+	migrate_leds "^youku-yk1:=yk1:"
+	;;
+zyxel,keenetic-omni)
+	migrate_leds "^kn_rc:=keenetic-omni:"
+	;;
+zyxel,keenetic-omni-ii)
+	migrate_leds "^kn_rc:=keenetic-omni-ii:"
+	;;
+zyxel,keenetic-viva)
+	migrate_leds "^kng_rc:=keenetic-viva:"
+	;;
+esac
```

Comment:
Am I mistaken, or are those the other renames only introduced by me? So, for your patch you would only need the case with wndr3700-v5?

Reply:
You are right, after importing this file from your patch I missed to cut it down.